Libraries/xcexecution/Headers/xcexecution/NinjaExecutor.h

```diff
@@ -11,6 +11,7 @@
 #define __xcexecution_NinjaExecutor_h
```

Comment:
Overall this commit LGTM.  I know little about ninja, though, so my LGTM is more like "I trust you".